pom.xml

```diff
@@ -9,7 +9,7 @@
     <version>1.0-SNAPSHOT</version>
 
     <properties>
-        <jdk.version>17</jdk.version>
+        <jdk.version>11</jdk.version>
```

Comment:
any reason to do that?

Reply:
cuz, IntelliJ IDEA 11 is installed on my laptop, I had to refactor it a bit. Should I leave it as it is or upgrade my platform?

Reply:
upgrade it - you will need it for future tasks